### PR TITLE
Gha update

### PR DESCRIPTION
**Pull request by hansvancalster**
Rework GHA's and contributing guidelines in order to minimize overhead.

### CODE REVIEW

File: .github/workflows/check_protocol.yml
Comment (ElsLommelen) [inline .github/workflows/check_protocol.yml:11]: Is this condition only needed for `sfp`-protocols?  Otherwise you need to use parentheses, as without parenthesis, the normal order of operations will be respected, so `and` will be performed first, and `or` afterwards.

File: .github/workflows/update_news_zenodo.yml
Comment (ElsLommelen) [inline .github/workflows/update_news_zenodo.yml:9]: Same remark as above: `and` operations have priority over `or` operations, so for protocols other than `sfp`, the GHA will also run if the review state is other than 'approved' without using parentheses.

File: .github/workflows/update_news_zenodo.yml
Comment (ElsLommelen) [inline .github/workflows/update_news_zenodo.yml:5]:
```suggestion
    types: [submitted]
    branches:
     - main
```

**Review by ElsLommelen — COMMENTED** [review]
I guess it should be ok after changing the remarks on adding parentheses, but as variable names inside GHA are different for different types of GHA (e.g. referring to the current branch name can be done by `GITHUB_HEAD_REF` in GHA `on pull request`, but a different variable name can be needed to refer to the branch in other GHA such as `on approval`), you may want to test it to be sure.  Or just keep in mind that it can go wrong when doing the first approval.  ;-)

File: .github/workflows/update_news_zenodo.yml
Comment (hansvancalster) [reply .github/workflows/update_news_zenodo.yml:9]: Good catch, will fix this

File: .github/workflows/update_news_zenodo.yml
Comment (ElsLommelen) [reply .github/workflows/update_news_zenodo.yml:5]: I found it in an older version of testprotocol, so we used it that way before we decided on putting both dockers together in the same GHA.  But I don't know if it worked, I guess I haven't tested it (other than the fact that it caused no problems in a PR to main).
Anyway, I leave it up to you to decide if it is necessary to add it: will there ever be a branch named after a protocol that will not be merging to the main?  And is it a problem if a GHA tries to add a NEWS.md file in this case?  ;-)

**Review by ElsLommelen — APPROVED** [review]

File: .github/workflows/update_news_zenodo.yml
Comment (hansvancalster) [reply .github/workflows/update_news_zenodo.yml:5]: See [48608cf](https://github.com/inbo/protocolsource/pull/81/commits/48608cf0f0f814f0cb9837245c6127d09a97a08d), the extra `&& github.event.pull_request.base.ref == 'main'` should cover this.

**Pull request merged**
